data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/metrics/Metrics.java

```diff
@@ -70,7 +70,7 @@ public class Metrics {
   // bounded, so we keep the resource usage under control.
   // We might want to provide configs to make it bigger than a single thread but a single thread
   // to start with is good enough for now.
-  public static final ExecutorService meterBinderExecutor = Executors.newSingleThreadExecutor();
+  public static final ExecutorService meterBinderExecutor = Executors.newFixedThreadPool(2);
```

Comment:
Why 2 and not 3/4/5?

Reply:
No need for this, I'll revert this change